README.md

```diff
@@ -1,2 +1,56 @@
-# Telegraf-Output-Apache-Arrow-Flight
-External Telegraf plugin for writing time series to Apache Arrow Flight
+# Apache Arrow Flight External Output Plugin
+
+This plugin writes to [ModelarDB-RS](https://github.com/ModelarData/ModelarDB-RS) via the [Apache Arrow Flight protocol](https://arrow.apache.org/docs/format/Flight.html).
```

Comment:
I tend to agree. It just seemed very premature to already call it a "general purpose" because it was so specific to our use case. I have made the suggested changes, but I don't see us getting any contributions until the plugin has been made totally general purpose and made internal via a [pull request](https://github.com/influxdata/telegraf/blob/master/CONTRIBUTING.md#contributing-code).

Reply:
I agree that we should aim the documentation and plugin at being general purpose and might as well do this now. As long as we don't make any contributions to Telegraf before it is completely done.

Reply:
With the exception of a few minor things the updated README is fine for me. @CGodiksen please resolve this discussion if agree.